docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -0,0 +1,144 @@
+
+The following comparison operators are available for numeric types (`Int`, `Float`, xref::type-definitions/types.adoc#type-definitions-types-bigint[`BigInt`])
+
+* `_LT`
+* `_LTE`
+* `_GTE`
+* `_GT`
+
+NOTE: Filtering on xref::type-definitions/types.adoc#type-definitions-types-temporal[Temporal Types] and xref::type-definitions/types.adoc#type-definitions-types-spatial[Spatial Types]: is not yet supported.
+
+=== String comparison
+
+The following case-sensitive comparison operators are only available for use on `String` and `ID` types:
+
+* `_STARTS_WITH`
+* `_NOT_STARTS_WITH`
+* `_ENDS_WITH`
+* `_NOT_ENDS_WITH`
+* `_CONTAINS`
+* `_NOT_CONTAINS`
+
+=== Array comparison
+
+The following two comparison operators are available on non-array fields, and accept an array argument:
+
+* `_IN`
+* `_NOT_IN`
+
+Conversely, the following operators are available on array fields, and accept a single argument:
+
+* `_INCLUDES`
+* `_NOT_INCLUDES`
+
+These four operators are available for all types apart from `Boolean`.
+
+== Usage
+
+Basic filtering of events can be done with the `where` parameter. This lets you filter on top-level properties of the created nodes.
+Only events matching these properties and type will be returned to this subscription.
+
+Considering the following type definitions:
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    genre: String
+    averageRating: Float
+}
+----
+below are some example of how filtering can be applied when creating a subscription.
+
+=== Create
+We can filter our movies by their genre:
+
+[source, graphql, indent=0]
+----
+movieCreated(where: {genre: "Drama"}) {
+    createdMovie {
+        title
+    }
+}
+----
+
+This way, only newly created movies with the genre `"Drama"` will trigger events to this subscription.
+
+NOTE: `Where` will only filter by properties set at the moment of creation.
+
+=== Update
+We can filter our movies with the average rating bigger than 8:
+
+[source, graphql, indent=0]
+----
+movieUpdate(where: {averageRating_GT: 8}) {
+    updatedMovie {
+        title
+    }
+}
+----
+
+This way, we will only receive events triggered by movies with the average rating bigger than 8 being modified.
+
+[source, graphql, indent=0]
+----
+mutation {
+    makeTheMatrix: createMovies(input: {title: "The Matrix", averageRating: 8.7}) {
+        title
+        averageRating
+    },
+    makeResurrections: createMovies(input: {title: "The Matrix Resurrections", averageRating: 5.7}) {
+        title
+        averageRating
+    },
+}
+
+mutation {
+    updateTheMatrix: updateMovie(
+        where: {title: "The Matrix"}
+        update: {averageRating: 7.9}
+    ) {
+        title
+    },
+    updateResurrections: updateMovie(
+        where: {title: "The Matrix Resurrections"}
+        update: {averageRating: 8.9}
+    ) {
+        title
+    }
+}
+----
+
+Therefore, given the above subscription, these GraphQL operations will only be triggered for `"The Matrix"` movie.
+
```

Comment:
Maybe we should mention that we are comparing with the properties **before** the update